### PR TITLE
auto deploy to github pages

### PR DESCRIPTION
**Pull request by charlierudolph**
@jbpros @mattwynne @aslakhellesoy

### CODE REVIEW

File: scripts/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+rsync -a release/ example/
+mkdir -p example/jquery/dist
+cp node_modules/jquery/dist/jquery.min.js example/jquery/dist/jquery.min.js
+rm example/server.js
+
+# Publish to gh-pages
+cd example
+git init
+git config user.name "Travis CI"
+git config user.email "charles.rudolph@originate.com"
```
Comment (charlierudolph) [inline scripts/deploy.sh]: Can we create a new cucumber account (unless you have one already) who can be used exclusively for this sort of deployment?

**Comment by mattwynne** [comment]
I'm missing context @charlierudolph - what kind of account, with what service?

**Comment by mattwynne** [comment]
An email address do you mean?

**Comment by charlierudolph** [comment]
Auhh sorry, I meant a github account (which I believe needs to be tied to an email). So I was thinking of something like github username `cucumber-bot` with an email `cucumber-bot@cucumber.io`

**Comment by jbpros** [comment]
cucumbot :D

**Pull request closed without merging**